test/embedded/classes-no-stdlib.swift

```diff
@@ -0,0 +1,32 @@
+// RUN: %target-swift-emit-ir %s -parse-stdlib -enable-experimental-feature Embedded -target arm64e-apple-none | %FileCheck %s
+
+// TODO: investigate why windows is generating more metadata.
+// XFAIL: OS=windows-msvc
```

Comment:
No investigation needed. This is because swift compiler sources doesn't run on windows.

Reply:
removed!